### PR TITLE
Besu network support

### PR DESCRIPTION
**Pull request by kidambisrinivas**
## Motivation

- Currently there is no support for running Private chains using Besu client node.

## Solution

- Explore how to add support for Besu client node and find the correct docker settings of a working solution
- Find the right settings for running Besu bootnode and RPC node

### CODE REVIEW

File: blockchain/config.go [hunk 1]
```diff
@@ -51,6 +51,8 @@ type EVMNetwork struct {
 	HTTPURLs []string `envconfig:"evm_http_urls" default:"http://example.url" toml:"evm_http_urls" json:"evm_http_urls"`
 	// True if the network is simulated like a geth instance in dev mode. False if the network is a real test or mainnet
 	Simulated bool `envconfig:"evm_simulated" default:"false" toml:"evm_simulated" json:"evm_simulated"`
+	// Type of chain client node. Values: "none" | "geth" | "besu"
+	SimulationType string `envconfig:"evm_simulation_type" default:"false" toml:"evm_simulation_type" json:"evm_simulation_type"`
```
Comment (kalverra) [inline blockchain/config.go:55]:
```suggestion
	SimulationType string `envconfig:"evm_simulation_type" default:"geth" toml:"evm_simulation_type" json:"evm_simulation_type"`
```

**Review by kalverra — CHANGES_REQUESTED** [review]

File: blockchain/config.go (hunk 1 above)
Comment (kidambisrinivas) [reply blockchain/config.go]: Fixed this

File: docker/test_env/non_dev_besu.go [hunk 2]
```diff
@@ -0,0 +1,409 @@
...
+	"github.com/google/uuid"
+	"github.com/pkg/errors"
+	"github.com/smartcontractkit/chainlink-testing-framework/blockchain"
+	"github.com/smartcontractkit/chainlink-testing-framework/utils/templates"
+	tc "github.com/testcontainers/testcontainers-go"
+	tcwait "github.com/testcontainers/testcontainers-go/wait"
+)
+
+const (
+	BESU_IMAGE = "hyperledger/besu:latest"
```
Comment (kalverra) [inline docker/test_env/non_dev_besu.go:25]: Do they have a `stable` or some similar tag?

Comment (kidambisrinivas) [reply docker/test_env/non_dev_besu.go:26]: Fixed this

**Review by kalverra — APPROVED** [review]

**Pull request merged**